cmd/utilities/utilities.go

```diff
@@ -81,6 +81,15 @@ func GenerateSitemap(cfg *config.Config, commandline *FlagFields) {
 		}),
 	)
 
+	return generator, nil
+}
+
+func GenerateSitemap(cfg *config.Config, commandline *FlagFields) {
+	generator, err := createCliSitemapGenerator(cfg, commandline)
+	if err != nil {
+		panic(err)
```

Comment:
We should not use panic if we can help it. It is better to give a reason for the error and/or return it so that it can be handled by the calling function.
```suggestion
		fmt.Println("Error creating sitemap generator", err.Error())
		return
```